File: AircraftSimulatorsControl/DAL/Entity/Report.cs
namespace AircraftSimulatorsControl.DAL.Entity
{
    public class Report : IEntity
    {

        public int ReportId { get; set; }

        public DateTime ReportDateTime { get; set; }

        public List<SimulatorStatus> SimulatorStatuses { get; set; }

    }
}

File: WebServer/src/main/java/org/server/dal/model/Report.java
package org.server.dal.model;

import lombok.Data;

import javax.persistence.*;
import java.time.LocalDate;
import java.util.Date;
import java.util.List;

@Table(name = "report")
@Entity
@Data
public class Report implements DatabaseEntity {

    @Id
    @Column(name = "id", unique = true)
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int id;

    @Column(name = "report_date_time")
    private LocalDate reportDateTime;

    @OneToMany(mappedBy = "report", cascade = CascadeType.ALL, orphanRemoval = true, fetch = FetchType.LAZY)
    private List<SimulatorStatus> simulatorStatuses;

    @Override
    public String toString() {
        return "Report{" +
                "id=" + id +
                ", reportDateTime=" + reportDateTime +
                '}';
    }
}
